### ISSUE TITLE
Can't add detailed location (full address) to a Listing

### ISSUE THREAD

**Issue by nmulnick** [marketplacekit/marketplacekit#90]
Hello,

This comment is for the hosted version where the php is not editable. Can you add the ability to store and display the full address for a listing so that via modifying the theme users have the ability to customize how location/address is displayed. For my use case, it is important to be able to display the entire address for a property.

When a user creates a listing, they can modify the "Area" (city) field to contain additional detail by changing the auto-populated text, but that is not intuitive as it doesn't auto-populate the full address from the location autocompleted for the map. I have modified the create listing location to use:
`addressFormat: 'formatted_address',`
This allows the full address to display as the location above the map. I would correspondingly like to have the "Area" field (or another field I insert) display the full address or at least be able to display the fields 'addressLine1' and 'addressLine2'. I can then update the browse and listing pages to display the full address. Currently, I don't believe this is possible because the Model/Listings.php file does not include these fields. Can you update to include or advise if there is another way to accomplish this?

To summarize, I want the following to be possible. 
1) When a user creates a listing, enters a location, the full address is auto-populated in the "Area" field. User can then modify what is displayed for the address if the Google Autocomplete isn't accurate.
2) I want to be able to display the full address (including street number and name) on a listing and on the browse listing pages.

Thanks! This application is wonderful and has amazing potential. Hoping that the above is possible so that we can use this for our use case.
